webapp/src/Twig/TwigExtension.php

```diff
@@ -174,6 +175,16 @@ public function printtimediff(float $start, ?float $end = null): string
         return Utils::printtimediff($start, $end);
     }
 
+    public function printremainingminutes(float $start, float $end): string
+    {
+        $minutesRemaining = round(($end - $start)/60, 0);
```

Comment:
Should this be floor instead of round?

Reply:
I first had put floor, then changed it round, so clearly don't have a strong opinion.